src/pages/billing/List.js

```diff
@@ -257,7 +286,9 @@ export const BillingList = _.flow(
             this.loadProjects()
           }
         }),
-        !!selectedName && billingProjects && h(ProjectDetail, { key: selectedName, project: _.find({ projectName: selectedName }, billingProjects) }),
+        noBillingProjects && hasFreeCredits && freeCreditsMessage,
```

Comment:
This could be a nice spot for a `Utils.cond`

Reply:
Good suggestion! I'll put that in